tech-committee-charter.md

```diff
@@ -1 +1,110 @@
-# Technical Committee Charter
\ No newline at end of file
+# Technical Committee Charter
+## Guiding Principle
+
+The OpenFeature project will operate transparently, collaboratively, ethically, and in alignment with the interests of OpenFeature technology end-users. Project proposals, timelines, and status must not merely be open, but also easily visible to and discoverable by outsiders.
+
+## Evolution of OpenFeature Governance
+
+Most large, complex open source communities have both a business and a technical governance model. OpenFeature has a project-level Governance Committee, a project-level Technical Committee, and is a Sandbox project in the Cloud Native Computing Foundation.
+
+This OpenFeature Technical Committee Charter reflects the scope and expectations of the Technical Committee ("TC") with the maintainers, the consumers, and the ecosystem of the project. The charter will not be perfect, and so has a simple amendment process – a TC member proposes changes to be discussed and voted on by the full TC.
+
+## Establishment of the Technical Committee
+
+TC memberships are not time-limited. There is no maximum size of the TC. The size is expected to vary in order to ensure adequate coverage of important areas of expertise, balanced with the ability to make decisions efficiently. The TC must have at least four members and is recommended to have an odd number of members for tie-breaking purposes.
+
+TC membership is developed through leadership in the project and was bootstrapped initially by selection of most prolific contributors in the early stages of the project. The TC may add additional members to the TC by the [election process](#election-of-tc-members) described below. The TC members are expected to be active leaders in the project. A TC member may be removed from the TC by voluntary resignation, by a super-majority TC vote similar to the [election process](#election-of-tc-members), or in accordance with the participation rules described below.
+
+Changes to TC membership should be posted in the agenda document, and may be suggested as any other agenda item.
+
+### No Over-Representation
+
+No more than one-fourth (25%) of the TC members may be affiliated with the same employer (in the event of confusion or concern, the OpenTelemetry TC will defer to the CNCF definition of "same employer"). If removal or resignation of a TC member, or a change of employment by a TC member, creates a situation where more than one-fourth of the TC membership shares an employer, then the situation must be immediately remedied by the election of a new TC member, the resignation or removal of one or more TC members affiliated with the over-represented employer(s).
```

Comment:
Looks like some references to OpenTelemetry still remain.

Reply:
thank you, should be fixed now